templates/zerver/help/add-emoji.md

```diff
@@ -1,22 +1,19 @@
 # Emoji
 
-Emoji allow you to add a little character and some fun to your
+Emoji allows you to add a little character and some fun to your
 messages! This guide shows you how to send emoji by using Zulip’s
 built-in emoji menu, pasting Unicode, or typing the emoji code.
 
 ### Using the emoji menu
-1. Start by composing a [new message](/help/#posting-and-replying)
+1. Start by composing a [new message](/help/#reply-to-a-message).
 2. Click the smiley face under the left edge of the message box.
 3. Click any emoji to include it in your messsage.
 
-![emoji-menu](/static/images/help/emoji-menu.png)
-
-
 ### Unicode
 Zulip fully supports Unicode. To use Unicode in your message:
 
-1. Copy any emoji you would like to include onto your clipboard. For example:😀.
-2. Compose a [new message or reply](/help/#posting-and-replying)
+1. Copy any emoji you would like to include onto your clipboard. For example, 😀.
+2. Compose a [new message or reply](/help/#reply-to-a-message).
 3. Paste the emoji into the message box.
 
 **Note**: You can browse the
```

Comment:
Change this to tip. Please refer to the style guide https://zulip.readthedocs.io/en/latest/README.html#style-guide